plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/transport/IngestActionFilter.java

```diff
@@ -92,6 +92,7 @@ void processIndexRequest(String action, ActionListener listener, ActionFilterCha
             public void executed(IngestDocument ingestDocument) {
                 if (ingestDocument.isModified()) {
                     indexRequest.source(ingestDocument.getSource());
+                    // uboness: don't we also need to update the metadata (index, id, etc...)?
```

Comment:
good point, we will once we have the processor that updates them, which is being worked on.